swingset/src/main/java/com/nqadmin/swingset/SSComboBox.java

```diff
@@ -71,14 +71,19 @@
  * <b>Warning. This combobox may use GlazedLists and an item is automatically
  * inserted when {@link #getAllowNull()} is true. Do not use methods
  * that are based on index in the list, unless you're sure...</b>
+ * 
  * {@link #setSelectedItem(java.lang.Object) setSelectedItem(SSListItem)}
  * and {@link #getSelectedItem() getSelectedItem()}
  * are the correct techniques.
- * There are a variety of method to change the current combobox item
- * such as
- * {@link #setSelectedMapping(java.lang.Object) setSelectedMapping(Integer)}.
+ * 
+ * There are a variety of methods to change the current combobox item
+ * such as:
+ * {@link SSBaseComboBox#setSelectedMapping(java.lang.Object) setSelectedMapping(Integer)}
+ * {@link SSBaseComboBox#setSelectedOption(java.lang.Object) setSelectedOption(String)}
  * <p>
- * Notice that {@link #getSelectedMapping() }/{@link #getSelectedValue() }
+ * Notice that:
+ * {@link #setSelectedValue(Integer)} and 
+ * {@link SSBaseComboBox#setSelectedMapping(java.lang.Object) setSelectedMapping(Integer)}
  * return null in two situations related to {@link #getAllowNull() }
```

Comment:
That would **get**, not *set*.

Reply:
Operator error. Fix enroute.

Reply:
Fixed it before I was done reviewing. What confidence.